packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -17,8 +17,7 @@ const AdvertiserNameStats = () => {
     const { data: advertiserStats, isLoading } = useAdvertiserStats();
     const { isMobile } = useDevice();
 
-    // TODO: Use Skeleton loader here
```

Comment:
i've removed a lot of the loaders in my-profile actually 😅 cause of our current behaviour in V1, we just load the entire page, I think if we want to implement all these loaders, we need design team to come up with skeleton loaders for the page

Reply:
Yeah we actually don't need these loaders. It's fine to remove these in your PR.

Reply:
sureee 👍